src/components/shared/CurrencyInput/CurrencyInput.tsx

```diff
@@ -131,7 +133,7 @@ const CurrencyInput: FC<CurrencyInputProps> = props => {
           {maxAmount && (
             <ButtonWrapper onClick={onMaxAmountClick} disabled={!onMaxAmountClick}>
               <div className="raft__currencyInput__maxAmountValue">
-                <Icon variant="wallet" size="tiny" />
+                {showWalletIcon && <Icon variant="wallet" size="tiny" />}
```

Comment:
because sometimes we want to show max amount without wallet icon. for example max amount for withdraw is not wallet balance, it's user current savings.